CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contribution guidelines
+
+Ongoing development on this project is managed by this project board: https://github.com/neontribe/www/projects/6.
+
+## Getting started on development
+
+Any issues in the `To do` column are assumed to be important enough and have been prioritised.
+
+When contributing to this project please do the following:
+
+1. Pick something near the top of the to do column, which you feel you have the time and confidence to tackle.
+   - We expect you to learn while tackling issues, please don't be put off by a tasks' seeming complexity.
+   - If you need more information please follow up with the person who spawned the task.
+   - Check the [Definition of done](#definition-of-done) for the ticket
+1. Assign yourself and move the ticket to the In progress column.
+1. Create a branch with a name to help match it to the ticket, e.g.: `307-contrib-docs`. _n.b._ [avoid slashes in branch names](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name/2527452#2527452)
+1. Work on the feature branch, make small commits which include the _reason_ for each commit's change.
+1. Create a pull request when you're happy with the changeset, and happy that the ticket passes the ticket's [Definition of done](#definition-of-done).
+   _n.b._ Pull requests are the pull requesters responsibility, when it recieves the request number
+1. Await code review, see [Review proces](#review-process)
+
+Please make sure that you keep you activity up-to-date:
+
+- only assign one ticket at a time to yourself
+- name a branch based upon the ticket
+- if the ticket needs to be put on hold
+
+---
+
+WIP:
+
+## Getting tickets ready to be worked on
+
+- weekly meeting to discuss open tickets
+- tickets should be left in the backlog or closed if vetoed
+- anyone can create issues
+- creating an issue involves writing up a compelling argument for a feature, perhaps pictures. Adding a definition of done is important, doesn't have to be acceptance criteria
+
+## Review process
+
+2 or more peers need to approve the commit, when reviewing check the code for errors and check that it achieves the definition of done for the ticket
```

Comment:
```suggestion
Two or more peers need to approve the commit, when reviewing check the code for errors and check that it achieves the definition of done for the ticket
```

Reply:
To me, these are in fact 3 separate points that can be split over three lines (would improve readability, too):
```
- Two or more peers need to approve the commit 
- When reviewing, check the code for errors 
- Check that it achieves the definition of done for the ticket
```